bilal-waheed/blockchain/cli-tool/preorder

bilal-waheed opened the pull request:
### Overview
1. Added bns preorder support.
2. Updated the Readme file accordingly.
3. Updated the folder structure to:
src
    /bns
    /transactions

bilal-waheed committed:
feat: add support for bns preorder transaction

what: bns name preorder transaction

why: to allow user to preorder names

impact: low

bilal-waheed committed:
Delete .DS_Store

bilalanees98 commented on README.md:
```diff
@@ -38,3 +38,18 @@ Enter the command with the following syntax
+
+Enter the command with the following syntax
+
+`npm start namePreorder [network - mainnet | testnet] [salt] [sender's private key] [name] [stx to burn]`
```
specify if the name here equals name+namespace or not.
Accordingly I think it would be better to take name and namespace as separate args. 
You will need too reflect these changes in the code as well

bilalanees98 reviewed (changes requested):
Please make changes according to the comments specified

bilal-waheed committed:
feat: add namespace CLI input

what: namespace input from CLI

why: to segregate the fully qualified name to name.namespace based on user input

impact: low

bilal-waheed committed:
Merge branch 'bilal/blockchain/cli-tool/preorder' of https://github.com/bilal-waheed/stacks into bilal/blockchain/cli-tool/preorder

bilalanees98 reviewed (approved):
Approved

bilal-waheed committed:
fix: minor import fix in dustTransaction.js
